docs/site/tutorials/authentication/Authentication-Tutorial.md

```diff
@@ -694,16 +722,16 @@ It searches through an injected user repository of type `UserRepository`.
 
 The `convertToUserProfile(user: User): UserProfile` function takes in a **user**
 of type `User` and returns a user profile of type
-[UserProfile](https://github.com/strongloop/loopback-next/blob/master/packages/authentication/src/types.ts).
+[UserProfile](https://github.com/strongloop/loopback-next/blob/master/packages/security/src/types.ts).
 A user profile, in this case, is the minimum set of user properties which
 indentify an authenticated user.
```

Comment:
typo: `indentify`

Reply:
Woah, nice catch. That's an old sentence as well. ;)